Extensions/Effects/displacement-pixi-filter.js

```diff
@@ -0,0 +1,22 @@
+gdjs.PixiFiltersTools.registerFilterCreator("Displacement", {
+  makePIXIFilter: function() {
+    var displacementSprite = new PIXI.Sprite.fromImage(
+      "res/displacement_map.png"
+    ); //Use a basic image bundle with GD for now, prefer use a customizable file set by user.
+    var displacementFilter = new PIXI.filters.DisplacementFilter(
+      displacementSprite
+    );
+
+    return displacementFilter;
+  },
+
```

Comment:
You must add updateBooleanParameters and updateDoubleParameters even if they are empty. I could maybe make them optional.